README.rst

```diff
@@ -74,17 +74,12 @@ Static Code Checkers
 
 You can ensure that code complies to Twisted `coding standards <https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html>`_::
 
-  $ tox -e lint   # run pyflakes to check syntax
-  $ tox -e black  # run Black to check against coding standard
+  $ tox -e lint   # run pre-commit to check coding stanards
   $ tox -e mypy   # run MyPy static type checker to check for type errors
 
-To automatically format code according to the coding standards::
+Or, for speed, use pre-commit directly::
 
-  $ tox -e black-reformat
-
-Or, for speed, specify exactly what to format::
-
-  $ tox -e black-reformat -- path/to/file.py
+  $ pipx run pre-commit run
```

Comment:
`tox -e lint -- run` does that already

Reply:
Them maybe suggest

```suggestion
  $ tox -e lint -- run
```

I would prefer `tox -e lint` to only run on changes since trunk.  and have `tox -e lint --all-files` to be used by CI.

Reply:
running via `tox` instead of `pipx` adds quite a bit of slowness - `pipx` here is more of a demo on how it could be installed.